clients/go/cmd/zbctl/internal/commands/root.go

```diff
@@ -144,6 +146,9 @@ func setSecurityParamsAsEnv() (err error) {
 	if caCertPathFlag != "" {
 		setEnv(zbc.CaCertificatePath, caCertPathFlag)
 	}
+	if overrideAuthorityFlag != "" {
+		setEnv(zbc.OverrideAuthorityEnvVar, overrideAuthorityFlag)
+	}
```

Comment:
❓ It's a bit unclear to me how this works. There is no setEnv function that I can see in the scope, yet it is also used by other lines. Do I understand correctly that this sets the `zbc.OverrideAuthorityEnvVar` to the value of `overrideAuthorityFlag` if `overrideAuthorityFlag` is set? It looks to me like some strange global variables situation 😕